code/modules/mining/machine_vending.dm

```diff
@@ -49,6 +50,7 @@
 		new /datum/data/mining_equipment("P-KA Cosmetic Super Chassis",	/obj/item/borg/upgrade/modkit/chassis_mod,							200),
 		new /datum/data/mining_equipment("P-KA Cosmetic Hyper Chassis",	/obj/item/borg/upgrade/modkit/chassis_mod/orange,					200),
 		new /datum/data/mining_equipment("Mining Hardsuit",				/obj/item/clothing/suit/space/hardsuit/mining,						2000),
+		new /datum/data/mining_equipment("Hardsuit Jetpack Upgrade",	/obj/item/tank/jetpack/suit,										1000),
```

Comment:
CE only item, cannot be crafted or found otherwise.

Reply:
I 100% remember when miners had this item, see no reason why it was removed